docusaurus/docs/iOS/guides/working-with-the-composer.md

```diff
@@ -0,0 +1,221 @@
+
+### Customizing the Suggestions UI
+The UI SDK provides a `ChatSuggestionsVC` component responsible for showing user suggestions and commands suggestions.
+
+#### Basic Customization
+In case you only want to change the styling of the container that shows both suggestions, you can override the `ChatSuggestionsVC`. Here is an example of a basic customization of the container component:
+```swift
+final class CustomChatSuggestionsVC: ChatSuggestionsVC {
+    override func setUpAppearance() {
+        super.setUpAppearance()
+
+        collectionView.layer.borderWidth = 1
+        collectionView.layer.borderColor = appearance.colorPalette.border.cgColor
+        collectionView.layer.cornerRadius = 10
+        view.layer.shadowOpacity = 0.0
+    }
+}
+
+components.suggestionsVC = CustomChatSuggestionsVC.self
+```
+Here we simply remove the shadow and customize the border style of the suggestions component.
+
+**Result:**
+
+| Before  | After |
+| ------------- | ------------- |
+| <img src={require("../assets/composer-suggestionsVC-before.png").default} width="100%"/> | <img src={require("../assets/composer-suggestionsVC-after.png").default} width="100%"/> |
+
+If you want more customizability and want to change the cells of the suggestions and how the data is formatted, the recommended way is to subclass the `ChatMentionSuggestionView` and `ChatCommandSuggestionView` components.
+<!--
+#### Example customization of the `ChatMentionSuggestionView`:
+TODO
+
+#### Example customization of the `ChatCommandSuggestionsView`:
+TODO
+--->
+
+#### Advanced Customization
+Finally, if you need to have complete control over how the suggestions are presented and not use our `ChatSuggestionsVC` component you can by overriding the following functions:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override func showMentionSuggestions(for typingMention: String, mentionRange: NSRange) {
+        // Add your custom implementation to show mention suggestions
+    }
+
+    override func showCommandSuggestions(for typingCommand: String) {
+        // Add your custom implementation to show command suggestions
+    }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+
+<!--
+### Adding New Types of Suggestions
+TODO (A good example is recognizing emojis, just like slack/discord.)
+--->
+
+### Disabling the Suggestions
+Your app might not want to support suggestions at all like the iMessage app, or want to support only one of the suggestions available. You can disable the suggestions feature by just returning `nil` on the following functions:
+```swift
+final class iMessageComposerVC: ComposerVC {
+    override func typingMention(in textView: UITextView) -> (String, NSRange)? {
+        // Don't show user suggestions
+        return nil
+    }
+
+    override func typingCommand(in textView: UITextView) -> String? {
+        // Don't show command suggestions
+        return nil
+    }
```

Comment:
We should explain that this should be disabled from channel type in dashboard, but it's possible like this too

Reply:
Oh, I just saw that the config logic is wrongly on `showCommandsFunction`, it should be here. But these are different things. You maybe still want the commands, but don't want the typing suggestions. This is showing how you disable only the typing suggestions, not the functionality of the commands, that should be with the config yes.